packages/create-next-app/package.json

```diff
@@ -14,9 +14,7 @@
   },
   "author": "Next.js Team <support@vercel.com>",
   "license": "MIT",
-  "bin": {
-    "create-next-app": "./dist/index.js"
-  },
+  "bin": "./dist/index.js",
```

Comment:
Just an artifact of Yarn 2, which automatically reformats manifests when it detects that some patterns are redundant (as another example, it would also remove the `dependencies` fields if they were empty objects).